volatility3/framework/symbols/windows/pdbutil.py

```diff
@@ -409,6 +409,12 @@ def symbol_table_from_pdb(
         _, symbol_table_name = cls._modtable_from_pdb(
             context, config_path, layer_name, pdb_name, module_offset, module_size
         )
+
+        if symbol_table_name is None:
+            raise exceptions.VolatilityException(
```

Comment:
Should probably be a `SymbolSpaceError` or something more specific, but otherwise looks good.

Reply:
That would require changing the `modtable_from_pdb` and also the current callers. Is that worth it?

Reply:
Why would it?  `SymbolSpaceError` inherits from `VolatilityError`, but people that want to catch *only* SymbolSpaceErrors can?

Reply:
changed both instances